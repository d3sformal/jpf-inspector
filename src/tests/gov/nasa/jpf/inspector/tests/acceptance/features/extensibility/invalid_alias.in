badbreak
badbreak2
hello && badbreak