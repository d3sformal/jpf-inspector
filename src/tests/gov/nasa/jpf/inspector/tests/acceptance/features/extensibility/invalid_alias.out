ERR: Parse error: no viable alternative at input 'ThisIsNotAValidHitCondition'
ThisIsNotAValidHitCondition
                           ^
ERR: The alias 'badbreak2' was expanded but its expansion could not be parsed.
The expansion was: 'create show breakpoint in this command "show" is not correct'.
Error: ERR: Parse error: mismatched input 'show' expecting TOKEN_BREAKPOINT.

Hello to you, too, dear user! The JPF Inspector command loop is working!
ERR: Parse error: no viable alternative at input 'ThisIsNotAValidHitCondition'
ThisIsNotAValidHitCondition
                           ^